### ISSUE TITLE
Yeti/bird monster forces special death animation on Lara

### ISSUE THREAD

**Issue by AkyV** [TokyoSU/Plugin_TokyoSU#37]
This alkready works with OCB. Must be told in OCB table and the readme.

**Comment by TokyoSU** [comment]
if i have the time :x 
current entity with ocb:
- TREX
- SPEAR_GUARDIAN
- YETI
PS:
bird monster not have special kill anim.

**Comment by AkyV** [comment]
No, previously I said that I can make the bird monster to represent a special Lara's death animation with using the animation ID as OCB.
And, as I already said here at another entry, crow/eagle also has OCB 1 for an immediate attack.

What are TREX/Spear Guardian OCBs?
